docs/src/notation.md

```diff
@@ -1,20 +1,18 @@
-# Notation on LIe Groups
+# Notation on Lie Groups
 
-For most of the notation used throughout this package, we follow [HilgertNeeb:2012](@cite)
+For most of the notation used throughout this package, we follow [HilgertNeeb:2012](@cite).
 
 | Symbol | Description | Also used | Comment |
 |:--:|:--------------- |:--:|:-- |
 | ``∘`` | a group operation | |
-| ``c:\mathcal G → \mathcal G, c_g`` | the conjugation map (with `g`) | |
+| ``c_g:\mathcal G → \mathcal G`` | the conjugation map (with `g`) | |
 | ``\mathrm{e}`` | identity element of a group | |
 | ``g, h, k`` | elements on a (Lie) group. Sometimes called points. | ``g_1, g_2, ...`` |
 | ``\mathfrak g`` | a Lie algebra | |
 | ``\mathcal{G}`` | a (Lie) group | |
-| ``λ_g: \mathcal G → \mathcal G`` | the left multiplication map ``λ_g(h) = g∘h`` | |
+| ``λ_g: \mathcal G → \mathcal G`` | the left group operation map ``λ_g(h) = g∘h`` | |
 | ``\mathcal M`` | a manifold | ``\mathcal M_1, \mathcal M_2,\ldots,\mathcal N`` | |
```

Comment:
Maybe instead of mentioning things defined in Manifolds.jl notation file we should just link to it, and here only provide new definitions?

Reply:
Fixed it.